packages/wonder-blocks-button/package.json

```diff
@@ -2,9 +2,11 @@
   "name": "wonder-blocks-button",
   "version": "0.0.1",
   "description": "",
-  "main": "index.js",
+  "main": "dist/index.js",
+  "source": "index.js",
```

Comment:
This is a hack so that webapp can compile wonder-blocks-button (et al) from source.  We'll need to update dev/tools/update-wonder-blocks-shims.js in webapp to use "source" instead of "main" since webapp compiles wonder-blocks for us.

Reply:
is there a way to tell webapp to not compile and instead use the already compiled artifact?  I know im missing some details why we wouldnt do that...  do we recompile all third party code?

Reply:
meh - ignore. I see the problem.  Let me think thru this for a minute. BRB

Reply:
I talked with @jeresig about that and we decided to maintain webapp's ability to use the source directly for the time being.  If we haven't used that ability in 6 months time then we'll re-evaluate.